amti/clis/preview.py

```diff
@@ -0,0 +1,100 @@
+"""CLI for running a simple web server to preview HIT"""
+
+import os
+import json
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from urllib.parse import urlparse
+
+import click
+import jinja2
+
+from amti import actions
+from amti import settings
+from amti import utils
+
+
+class MyServer(HTTPServer):
+    def __init__(
+        self,
+        server_address,
+        RequestHandlerClass,
+        template_path=None,
+        data_path=None,
+    ):
+        super().__init__(server_address, RequestHandlerClass)
+        self.template_path = template_path
+        self.data_path = data_path
+
+
+class MyHandler(BaseHTTPRequestHandler):
```

Comment:
Similarly to above, since there's only one handler, we can probably call this class `Handler`.